cmd/install/assets/hypershift_operator.go

```diff
@@ -142,10 +142,15 @@ type ExternalDNSDeployment struct {
 	Provider          string
 	DomainFilter      string
 	CredentialsSecret *corev1.Secret
+	TxtOwnerId        string
 }
 
 func (o ExternalDNSDeployment) Build() *appsv1.Deployment {
 	replicas := int32(1)
+	txtOwnerId := o.TxtOwnerId
+	if txtOwnerId == "" {
```

Comment:
Does it really make sense to use a UUID? If someone re-renders or applies, it will be different

Reply:
the external-dns [docs](https://github.com/kubernetes-sigs/external-dns/blob/master/README.md) mention this

> We strongly encourage you to use v0.5 (or greater) with --registry=txt enabled and --txt-owner-id set to a unique value that doesn't change for the lifetime of your cluster.

my understanding is that this value should not changed once external-dns runs on a cluster and being able to provide a fixed value helps with that. please let me know if i got this wrong.

Reply:
Yeah, exactly but it will if you re-run `hypershift install` multiple times without the `--txt-owner-id` account

Reply:
To me it makes sense to remove the UUID logic altogether. My change was trying to work around it because I did not want to modify existing behavior.

Reply:
Oh I missed the fact that you didn't add it. I was also in favor of removing it. If we just do that we will likely break CI though.